### ISSUE TITLE
Network options --offline and -C fails after new Miniconda installation with libmamba solver as default.

### ISSUE THREAD

**Issue by betofraus** [conda/conda-libmamba-solver#396]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

After installing Miniconda with libmamba solver as default (I did not have any other conda installation previously), 
I used the command 
`conda update --all --dry-run --offline`
and this is the error:

    FileNotFoundError: [Errno 2] No such file or directory: '/home/miniconda3/pkgs/cache/47929eba.info.json'

If I execute
`conda install python --dry-run`
such file is generated and --offline options works

Note: With the classic solver I do not have this issue.

### Conda Info

```shell
active environment : base
    active env location : /home/miniconda3
            shell level : 1
       user config file : /home/.condarc
 populated config files : /home/.condarc
          conda version : 23.10.0
    conda-build version : not installed
         python version : 3.11.5.final.0
       virtual packages : __archspec=1=x86_64_v4
                          __glibc=2.35=0
                          __linux=5.15.0=0
                          __unix=0=0
       base environment : /home/miniconda3  (writable)
      conda av data dir : /home/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/miniconda3/pkgs
                          /home/.conda/pkgs
       envs directories : /home/miniconda3/envs
                          /home/.conda/envs
               platform : linux-64
             user-agent : conda/23.10.0 requests/2.31.0 CPython/3.11.5 Linux/5.15.0-75-generic ubuntu/22.04.2 glibc/2.35 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.3
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Conda Config

```shell
==> /home/.condarc <==
solver: libmamba
```

### Conda list

```shell
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main
_openmp_mutex             5.1                       1_gnu
archspec                  0.2.1              pyhd3eb1b0_0
boltons                   23.0.0          py311h06a4308_0
brotli-python             1.0.9           py311h6a678d5_7
bzip2                     1.0.8                h7b6447c_0
c-ares                    1.19.1               h5eee18b_0
ca-certificates           2023.08.22           h06a4308_0
certifi                   2023.7.22       py311h06a4308_0
cffi                      1.15.1          py311h5eee18b_3
charset-normalizer        2.0.4              pyhd3eb1b0_0
conda                     23.10.0         py311h06a4308_0
conda-content-trust       0.2.0           py311h06a4308_0
conda-libmamba-solver     23.11.1         py311h06a4308_0
conda-package-handling    2.2.0           py311h06a4308_0
conda-package-streaming   0.9.0           py311h06a4308_0
cryptography              41.0.3          py311hdda0065_0
fmt                       9.1.0                hdb19cb5_0
icu                       73.1                 h6a678d5_0
idna                      3.4             py311h06a4308_0
jsonpatch                 1.32               pyhd3eb1b0_0
jsonpointer               2.1                pyhd3eb1b0_0
krb5                      1.20.1               h143b758_1
ld_impl_linux-64          2.38                 h1181459_1
libarchive                3.6.2                h6ac8c49_2
libcurl                   8.4.0                h251f7ec_0
libedit                   3.1.20221030         h5eee18b_0
libev                     4.33                 h7f8727e_1
libffi                    3.4.4                h6a678d5_0
libgcc-ng                 11.2.0               h1234567_1
libgomp                   11.2.0               h1234567_1
libmamba                  1.5.3                haf1ee3a_0
libmambapy                1.5.3           py311h2dafd23_0
libnghttp2                1.57.0               h2d74bed_0
libsolv                   0.7.24               he621ea3_0
libssh2                   1.10.0               hdbd6064_2
libstdcxx-ng              11.2.0               h1234567_1
libuuid                   1.41.5               h5eee18b_0
libxml2                   2.10.4               hf1b16e4_1
lz4-c                     1.9.4                h6a678d5_0
ncurses                   6.4                  h6a678d5_0
openssl                   3.0.12               h7f8727e_0
packaging                 23.1            py311h06a4308_0
pcre2                     10.42                hebb0a14_0
pip                       23.3            py311h06a4308_0
pluggy                    1.0.0           py311h06a4308_1
pybind11-abi              4                    hd3eb1b0_1
pycosat                   0.6.6           py311h5eee18b_0
pycparser                 2.21               pyhd3eb1b0_0
pyopenssl                 23.2.0          py311h06a4308_0
pysocks                   1.7.1           py311h06a4308_0
python                    3.11.5               h955ad1f_0
readline                  8.2                  h5eee18b_0
reproc                    14.2.4               h295c915_1
reproc-cpp                14.2.4               h295c915_1
requests                  2.31.0          py311h06a4308_0
ruamel.yaml               0.17.21         py311h5eee18b_0
setuptools                68.0.0          py311h06a4308_0
sqlite                    3.41.2               h5eee18b_0
tk                        8.6.12               h1ccaba5_0
tqdm                      4.65.0          py311h92b7b1e_0
truststore                0.8.0           py311h06a4308_0
tzdata                    2023c                h04d1e81_0
urllib3                   1.26.18         py311h06a4308_0
wheel                     0.41.2          py311h06a4308_0
xz                        5.4.2                h5eee18b_0
yaml-cpp                  0.8.0                h6a678d5_0
zlib                      1.2.13               h5eee18b_0
zstandard                 0.19.0          py311h5eee18b_0
zstd                      1.5.5                hc292b87_0
```

**Comment by jaimergp** [comment]
I'm not exactly sure what is supposed to happen with an offline update where there's no cache. What kind of behaviour are you expecting? Thanks for clarifying! 🙏

**Comment by betofraus** [comment]
I expect a similar behavior as with classic solver, or a warning message.

I used miniconda as an example. I found the issue while I was using an installer build with conda constructor. I made the installation with option -k (keep cache). If I use --offline with classic solver, there is no error. 
With libmamba solver I need to use a conda install command to generare the chache files, and after I can use the --offline option.
